src/pages/saksbehandling/revurdering/oppsummering/RevurderingsOppsummering.tsx

```diff
@@ -0,0 +1,189 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { useFormik } from 'formik';
+import { AlertStripeFeil, AlertStripeSuksess } from 'nav-frontend-alertstriper';
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import { Textarea } from 'nav-frontend-skjema';
+import { Innholdstittel } from 'nav-frontend-typografi';
+import React, { useCallback, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import { ApiError } from '~api/apiClient';
+import { sendTilAttestering } from '~api/revurderingApi';
+import * as revurderingSlice from '~features/revurdering/revurdering.slice';
+import { pipe } from '~lib/fp';
+import { useI18n } from '~lib/hooks';
+import * as Routes from '~lib/routes';
+import { Nullable } from '~lib/types';
+import yup from '~lib/validering';
+import VisBeregning from '~pages/saksbehandling/steg/beregningOgSimulering/beregning/VisBeregning';
+import { RevurderingSteg } from '~pages/saksbehandling/types';
+import { useAppSelector, useAppDispatch } from '~redux/Store';
+import { SimulertRevurdering, TilAttesteringRevurdering } from '~types/Revurdering';
+
+import messages from '../revurdering-nb';
+import sharedStyles from '../revurdering.module.less';
+import { VisFeilmelding } from '../VisFeilMelding';
+
+import styles from './revurderingsOppsummering.module.less';
+
+interface OppsummeringFormData {
+    tekstTilVedtaksbrev: Nullable<string>;
+}
+
+const schema = yup.object<OppsummeringFormData>({
+    tekstTilVedtaksbrev: yup.string().nullable().defined(),
+});
+
+const RevurderingsOppsummering = (props: { sakId: string; revurdering: SimulertRevurdering }) => {
+    const [hasSubmitted, setHasSubmitted] = useState<boolean>(false);
+    const intl = useI18n({ messages });
+    const dispatch = useAppDispatch();
+    const [sendtTilAttesteringStatus, setSendtTilAttesteringStatus] = useState<
+        RemoteData.RemoteData<ApiError, TilAttesteringRevurdering>
+    >(RemoteData.initial);
+
+    const { beregnOgSimulerStatus, revurderingsVedtakStatus } = useAppSelector((state) => state.revurdering);
+
+    const hentBrev = useCallback(
+        (fritekst: Nullable<string>) => {
+            if (RemoteData.isPending(revurderingsVedtakStatus)) {
+                return;
+            }
+
+            dispatch(
+                revurderingSlice.fetchRevurderingsVedtak({
+                    sakId: props.sakId,
+                    revurderingId: props.revurdering?.id ?? '',
+                    fritekst,
+                })
+            ).then((action) => {
+                if (revurderingSlice.fetchRevurderingsVedtak.fulfilled.match(action)) {
+                    window.open(action.payload.objectUrl);
+                }
+            });
+        },
+        [props.sakId]
```

Comment:
Må man ikke her også ha `revurderingsVedtakStatus` som avhengighet for at det skal bli riktig?

Reply:
Yes, tror det er riktig. good catch